terraform/windows.tf

```diff
@@ -1,3 +1,3 @@
 data "aws_ssm_parameter" "windows" {
-  name = "/aws/service/ami-windows-latest/Windows_Server-2019-English-Core-ContainersLatest"
+  name = "/aws/service/ami-windows-latest/Windows_Server-2019-English-Core-EKS_Optimized-1.23/image_id"
```

Comment:
just so I understand, ContainersLatest doesn't work even with `/image_id`? Surprising behavior in terms of backwards compatibility

Reply:
> ContainersLatest doesn't work even with /image_id

I think it no longer exists.